php.ini-development

```diff
@@ -356,6 +356,12 @@ disable_classes =
 ; http://php.net/realpath-cache-ttl
 ;realpath_cache_ttl = 120
 
+; Allows user to disable the "stat cache." By default php remembers the
+; results of the last call to stat() and lstat() and functions that call
+; it underneath.  Setting this to False disables that behaviour.
```

Comment:
Even if "False" is also allowed i think the preferred name is "Off" in the ini file

Reply:
Fixed.  And fixed the other one where I used "True" instead of "Off" which would be flatly wrong.